superset-frontend/src/explore/components/useExploreAdditionalActionsMenu/DashboardsSubMenu.test.tsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { render, screen, waitFor } from 'spec/helpers/testing-library';
+import userEvent from '@testing-library/user-event';
+import { Menu } from 'src/components/Menu';
+import DashboardItems from './DashboardsSubMenu';
+
+const asyncRender = (numberOfItems: number) =>
+  waitFor(() => {
+    const dashboards = [];
+    for (let i = 1; i <= numberOfItems; i += 1) {
+      dashboards.push({ id: i, dashboard_title: `Dashboard ${i}` });
+    }
+    render(
+      <Menu openKeys={['menu']}>
+        <Menu.SubMenu title="Dashboards added to" key="menu">
```

Comment:
The submenu label seems misleading, as it's not conveying that the entries are actions that open the dashboards in a new tab. I don't have a lot of great alternatives unfortunately, but maybe something like "Chart on <number> dashboards", where `<number>` is the number of dashboards on which the chart is featured?

Reply:
@kasiazjc @rusackas @lauderbaugh WDYT?

Reply:
We actually were struggling with the right label and we chose to proceed with this one as we have filter and column that are named like this in CRUD views - so at least it's consistent everywhere. 
I honestly don't know if this would be a better option, so waiting for @rusackas and @lauderbaugh to share their opinions 🙃

Reply:
@kasiazjc got it - makes sense and probably not something we should get hung up about (we can always change this later)